packages/jwt-verifier/lib.d.ts

```diff
@@ -9,9 +9,9 @@
  *
  * See the License for the specific language governing permissions and limitations under the License.
  */
-export = OktaJwtVerifier;
-declare class OktaJwtVerifier {
-  constructor(options: OktaJwtVerifier.VerifierOptions);
+export default class OktaJwtVerifier {
```

Comment:
I think this needs to be `export = OktaJwtVerifier;` since the export in lib.js is `module.exports = OktaJwtVerifier;`

Ref: https://www.typescriptlang.org/docs/handbook/modules.html#default-exports